src/assets/stylesheets/2d-hud.scss

```diff
@@ -61,7 +61,7 @@
       height: 0;
       border-left: 5px solid transparent;
       border-right: 5px solid transparent;
-      border-bottom: 5px solid $bright-blue;
+      border-bottom: 5px solid $notice-background-color;
```

Comment:
Why are some of these `var` and others scss variables?

Reply:
because the `var` ones are set by the light/dark theme completely (ie, there are no defaults if unspecified) then it seemed more DRY to avoid an unnecessary macro layer. the other ones have default values so to DRY up references to those defaults I made them macros.